src/content/docs/logs/get-started/logging-best-practices.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: Logging best practices guide
+tags:
+ - New Relic solutions
+ - Best practices guides
+ - Logs
+ - Logging
+translate:
+ - jp
+metaDescription: 'Best practices for using New Relic logs'
+redirects:
+---
+
+import parsingExample from 'images/logs-parsing-example.png'
+import image from 'images/image.png'
+
+Welcome to the New Relic logging best practices guide. Here you'll find recommendations for:
+
+- [Forwarding logs](#forwarding-logs)
+- [Data partitions](#partitions)
+- [Parsing logs](#parsing-logs)
+- [Drop filter rules](#drop-rules)
+- [New Relic log sizing](#sizing)
+- [Searching logs](#searching-logs)
+- [What's next?](#whats-next)
+
+## Forwarding logs [#forwarding-logs]
+
+* When forwarding logs, always use our [New Relic infrastructure agent](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent) and/or [APM agents](/docs/apm/new-relic-apm/getting-started/get-started-logs-context#agents) when possible. If you cannot use New Relic agents, use other supported agents (FluentBit, Fluentd, Logstash, etc.).
+
+Here are some Github example configurations for supported logging agents:
+   * [FluentBit examples](https://github.com/newrelic/fluentbit-examples) 
+   * [Fluentd examples](https://github.com/newrelic/fluentd-examples/)
+   * [Logstash examples](https://github.com/newrelic/logstash-examples)
+
+<Callout variant="warning">
```

Comment:
How do they edit the infra agent configs? If so, I'll add instructions or a docs link here.

Reply:
here's the infra log doc, which contains some details on this https://docs.newrelic.com/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/

Reply:
Hmm, that doc doesn't have any info on enabling/disabling log collection. Am I missing it on this page? Or if it's not there, is this something customers know how to do, so we don't need to add info on it?